docs/Tutorials/Create-IBFT-Network.md

```diff
@@ -0,0 +1,267 @@
+
+To obtain the address for Node-1, in the `Node-1` directory, use the [`public-key export-address`](../Reference/Pantheon-CLI-Syntax.md#public-key)
+subcommand to write the node address to the specified file (`nodeAddress1` in this example)
+
+```bash tab="MacOS"
+pantheon --data-path=Node-1-data-path public-key export-address --to=Node-1-data-path/nodeAddress1
+```
+
+```bash tab="Windows"
+pantheon --data-path=Node-1-data-path public-key export-address --to=Node-1-data-path\nodeAddress1
+```
+
+### 4. RLP Encode Extra Data 
+
+The `extraData` property in IBFT 2.0 genesis files is an RLP encoding of `[32 Bytes Vanity, List<Validators>, No Votes, Round=Int(0), 0 Seals]`. 
+
+Use the Pantheon subcommand `encode` to generate the `extraData`. 
+
+### 5. Create Genesis File 
+
+The genesis file defines the genesis block of the blockchain (that is, the start of the blockchain).
+The [IBFT 2.0 genesis file](../Consensus-Protocols/IBFT.md#genesis-file) includes the address of Node-1 as the initial validator in the
+data encoded in the `extraData` field.    
+
+All nodes in a network must use the same genesis file. 
+
+Copy the following genesis definition to a file called `ibftGenesis.json` and save it in the `IBFT-Network` directory: 
+
+```json
+{
+  "config":{
+    "chainId":1981,
+    "constantinoplefixblock": 0,
+    "ibft2": {
+      "blockperiodseconds": 2,
+      "epochlength": 30000,
+      "requesttimeoutseconds": 10
+     }
+    }
+  },
+  "nonce": "0x0",
+  "extraData": "<RLP Encoded Data>",
+  "difficulty": "0x1",
+  "mixHash": "0x63746963616c2062797a616e74696e65206661756c7420746f6c6572616e6365",
+  "timestamp":"0x5c51a607",
+  "alloc": {
+      "fe3b557e8fb62b89f4916b721be55ceb828dbd73": {
+        "privateKey": "8f2a55949038a9610f50fb23b5883af3b4ecb3c3bb792cbcefbd1542c692be63",
+        "comment": "private key and this comment are ignored.  In a real chain, the private key should NOT be stored",
+        "balance": "0xad78ebc5ac6200000"
+      },
+      "627306090abaB3A6e1400e9345bC60c78a8BEf57": {
+        "privateKey": "c87509a1c067bbde78beb793e6fa76530b6382a4c0241e5e4a9ec0a0f44dc0d3",
+        "comment": "private key and this comment are ignored.  In a real chain, the private key should NOT be stored",
+        "balance": "90000000000000000000000"
+      },
+      "f17f52151EbEF6C7334FAD080c5704D77216b732": {
+        "privateKey": "ae6ae8e5ccbfb04590405997ee2d52d2b330726137b875053c36d94e974d162f",
+        "comment": "private key and this comment are ignored.  In a real chain, the private key should NOT be stored",
+        "balance": "90000000000000000000000"
+      }
+   },
+  "number":"0x0",
+  "gasUsed":"0x0",
+  "parentHash":"0x0000000000000000000000000000000000000000000000000000000000000000"
+}
+```
+
+In `extraData`, include the [RLP encoded data that includes the initial validator](#4-rlp-encode-extra-data). 
+
+!!! example
+    
+    ```json
+    {
+      ...
+    "extraData":"UPDATE",
```

Comment:
If you use the 3 addresses already in the genesis it your give an extradata rlp string like : 
```suggestion
    "extraData":"0xf869a00000000000000000000000000000000000000000000000000000000000000000f83f94fe3b557e8fb62b89f4916b721be55ceb828dbd7394627306090abab3a6e1400e9345bc60c78a8bef5794f17f52151ebef6c7334fad080c5704d77216b732808400000000c0"
```

From the Json : 
```
[
"fe3b557e8fb62b89f4916b721be55ceb828dbd73",
"627306090abaB3A6e1400e9345bC60c78a8BEf57",
"f17f52151EbEF6C7334FAD080c5704D77216b732"
]
```

Reply:
Fixed